### ISSUE TITLE
Arguments can't have double quotes.

### ISSUE THREAD

**Issue by AngelCMHxD** [nwutils/create-desktop-shortcuts#63]
If at the time you create a shortcut, the argument you set has double quotes ("), a error will pop-up (It occurs in the VBScript "windows.vbs"). After some research, this error occurs because of how strings are defined in VBScript and how it handles escape characters (You can't use a backslash as escape character).

Example: 
```js
const shortcutsCreated = createDesktopShortcut({
    windows: {
        filePath: "Path\\to\\file.exe",
        arguments: `--arg "Hello World"`
    }
});
```

Error pop-up:
![image](https://user-images.githubusercontent.com/57822483/173953953-93af6cb2-cbd6-42ac-9119-fcbcde9f9640.png)

After seeing in which part of the windows.vbs file was the error, I noticed that the Line:Char are in the part were you define the IconLocation, this is due to VBScript thinking that the string ended before.

**Comment by AngelCMHxD** [comment]
@TheJaredWilcurt Issue isn't fixed but more weird. Now every `"` is replaced by a `\\`, looks like VBScript is actually getting the double-quotes, but is also *in some way* getting node.js escaping character (`\`), I think the solution could be replacing the double-quotes directly in the VBScript file.

In Code:
![image](https://user-images.githubusercontent.com/57822483/174417293-615b7eb9-e5be-416e-a59b-bf23b9d3e07a.png)

In Shortcut:
![image](https://user-images.githubusercontent.com/57822483/174417289-20f7ea00-7689-4a88-824d-155fe48ee844.png)

**Comment by TheJaredWilcurt** [comment]
Okay, VBScript is even more annoying than I realized. All double quotes are stripped out of arguments passed in. So the only recourse I have is to replace them with a keyword first, then swap the keyword with double quotes once in the VBScript.

Now that the `windows.vbs` file is holding more logic, I've added in an End-to-End test to verify that the shortcut it produces matches the inputs passed into the JS library's API. If this was in place before, then I would have caught this before doing a release. Okay, *should* be good this time. 🤞

Fixed in v1.9.0.
